incubator/hnc/docs/user-guide/how-to.md

```diff
@@ -478,6 +497,51 @@ kubectl get crds | grep .hnc.x-k8s.io | awk '{print $1}' | xargs kubectl delete
 kubectl delete -f https://github.com/kubernetes-sigs/multi-tenancy/releases/download/hnc-${HNC_VERSION}/hnc-manager.yaml
 ```
 
+<a name="admin-excluded-namespaces"/>
+
+### Excluding namespaces from HNC
+
+***The following instructions are only required for HNC v0.8.x and higher***
+
+HNC installs a validating webhook on _all_ objects in your cluster. If HNC
+itself is damaged or inaccessible, this could result in all changes to all
+objects in your cluster being rejected, making it difficult to repair your
+cluster or even re-install HNC.
+
+
+In order to prevent HNC from damaging your cluster, you can exclude certain
+namespaces from some of HNC's webhooks. Excluded namespace cannot be the
+parent or child of any other namespace; any attempts to change the hierarchy of
+an excluded namespace, or create a subnamespace within it, will be rejected by
+HNC. However, the critical webhooks will not operate in the excluded namespace,
+protecting your cluster's stability.
+
+In order to exclude namespaces from HNC _before_ installing it:
+
+1. Add the `hnc.x-k8s.io/excluded-namespace` label with the value of `true` to
+   all critical namespaces. We recommend, at a minimum, applying this label to
+   `kube-system`, `kube-public`, and `kube-node-lease`.  Note that the HNC
+   manifests already include this label on `hnc-system` itself.
+2. Ensure that all the namespaces you have excluded are also listed in the
+   [argument list](#admin-cli-args) with the option `--excluded-namespace`. By
+   default, the HNC manifests include all the critical system namespaces listed
+   above, but you can exclude any namespace you like.
+3. Apply the HNC manifest.
+
+To exclude an additional namespace from HNC _after_ it has been installed,
+follow these steps:
+
+1. Add the namespace to the list of `--excluded-namespace` [command line
```

Comment:
Add "and apply the manifest"?

Reply:
I've added directions in the linked section about how to redeploy  - e.g., you don't necessary have to modify the manifest (e.g. you can just say `kubectl edit`). So I'd rather leave this as-is.